### ISSUE TITLE
[openSUSE] remove firewalld associated code

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2024]
Thanks to @Psykar in the following forum thread for highlighting this openSUSE related issue. As from Leap 15.1 (currently beta) firewalld has been dropped as part of a default openSUSE server install. As we currently have code that simply disables this services it is fitting that we consider removing this code, given our ongoing plans to transition over to an openSUSE base for our upstream distro. This would in turn help to reduce our install image size and consequent code maintenance surface.

A further related consideration is the removal of this package as an rpm dependency.

**Comment by Psykar** [comment]
Hrm... I'm not 100% it was removed from the default install actually - rather removed from the default docker image.

**Comment by phillxnet** [comment]
@Psykar Yes I didn't note down where I got this from unfortunately but we do have from Mr Richard Brown himself @sysrich:

https://lists.opensuse.org/opensuse-factory/2018-11/msg00142.html

"_Therefore I've proposed the following PR to both Leap 15.1 and
Tumbleweed's installers_"
...
"As we now have clear "Server" and "Transactional Server" system roles,
the firewall & sshd configuration for those roles will now be
optimised by default

- sshd will be enabled by default
- firewalld will be disabled by default

I've discussed logic of disabling the firewall was discussed at length
with a number of people, especially our Leap release manager Ludwig
who's opinion on security I consider very highly.
"

So my wiki notes on our openSUSE move seem to be in line; mostly:
https://forum.rockstor.com/t/opensuse-dev-notes-and-status/5662
"
ssh enable (will be default come Leap 15.1 on server install)
ssh port will be open (again Leap 15.1 will disable firewall on server install)
"

My intention is to release Rockstor based on either of these roles from upstream, probably straight server initially and later move to the more ideal transactional but that will need more work on our part re our Web-UI initiated updates etc. 

But as per your docker image finding it may be from the following 15.1 Release Notes:
https://doc.opensuse.org/release-notes/x86_64/openSUSE/Leap/15.1/

"1.2 Minimal System Installation

The minimal system installation lacks certain functionality that is often taken for granted:

- It does not contain a software firewall front-end. You can install the package firewalld additionally.
- It does not contain a YaST. You can install the pattern patterns-yast-yast2_basis additionally.
"

So we are, via successive approximation, finding out way here.

Thanks for you input on these details by the way. Tricky to keep everything in line with moving targets / defaults. But we try; and it comes with the area we are in :) . I have updated my developer wiki with the above references and will edit this issue's text.

**Comment by sysrich** [comment]
While what I said was true when I said.. things have changed since then

https://github.com/yast/skelcd-control-openSUSE/pull/166
https://github.com/yast/skelcd-control-openSUSE/pull/167

firewalld is once again enabled by default on openSUSE's server system roles
